javax/src/main/java/org/jboss/ejb/client/SystemProperties.java

```diff
@@ -25,7 +25,7 @@
 final class SystemProperties {
 
     static final String DISCOVERY_ADDITIONAL_NODE_TIMEOUT = "org.jboss.ejb.client.discovery.additional-node-timeout";
-    static final String DISCOVERY_BLACKLIST_TIMEOUT = "org.jboss.ejb.client.discovery.blacklist.timeout";
+    static final String DISCOVERY_BLOCKLIST_TIMEOUT = "org.jboss.ejb.client.discovery.blocklist.timeout";
```

Comment:
I think the string value should remain unchanged, since it's used by the user arleady, and we don't want to break existing usages.  But the variable name can be updated.